runtime/processor/token_parser.cc

```diff
@@ -148,7 +148,6 @@ std::string TokenParser::reorder(const std::string& input) {
   for (auto& token : tokens) {
     output += token.string(orders) + " ";
   }
-  tokens.clear();
```

Comment:
Why delete tokens.clear() ?

Reply:
Cause `token_parser` is a local var now.

Reply:
okay